app/adapters/alma_adapter/availability_status.rb

```diff
@@ -105,6 +105,7 @@ def holding_status_from_item(alma_item)
 
     def to_h
       holdings.each_with_object({}) do |holding, acc|
+        # I want to find the in_temp_location = true and create a holding
```

Comment:
This is a note for the refactoring I have mentioned. I added it in the past; I would need to review again. For now I made a note and removed it from the codebase to avoid confusion. thank you.